### DISCUSSION TITLE
problem with scaling in AR

### DISCUSSION THREAD

**Discussion by Jackblack210** [google/model-viewer#2849] [Q&A]
hello, how can i modify the scale when i open it in AR view? It appears very small to me, it is a GLB file used.

Thanks.

![WhatsApp Image 2021-10-07 at 21 47 27](https://user-images.githubusercontent.com/67923535/136481640-4744e64a-babf-4799-9716-a431951d9e51.jpeg)

**Comment by ghost** [comment]
Do you control the file? The best way is fix the scale is on export, it looks like export may be set to 1 unit = 1cm and it should be 1m.

Alternatively, you can simply adjust the scale with the `scale` attribute [documented here](https://modelviewer.dev/docs/#entrydocs-scenegraph-attributes-scale).
